src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -29,6 +29,17 @@ Copyright (c) .NET Foundation. All rights reserved.
                                         Publish
  
     The main publish entry point.
+
+    Extension hooks for injecting custom targets:
+      - BeforeComputeFilesToPublishTargets
+      - AfterComputeFilesToPublishTargets
+      - BeforeILLinkTargets
+      - AfterILLinkTargets
```

Comment:
If we add public extension hooks, it could be used for any arbitrary targets by users.
We should have good testing to ensure this case is well supported.